src/l10n.json

```diff
@@ -256,5 +256,11 @@
     "comments.status.suspended": "Suspended",
     "comments.status.acctdel": "Account deleted",
     "comments.status.deleted": "Deleted",
-    "comments.status.reported": "Reported"
+    "comments.status.reported": "Reported",
+
+    "social.title": "Embed",
```

Comment:
I kept the "social" title because it's really more than just "embed", even without the social links.